content/en/policies/other/restrict_service_account.md

```diff
@@ -0,0 +1,50 @@
+---
+type: "docs"
+title: Restrict Service Account
+linkTitle: Restrict Service Account
+weight: 37
+description: >
+    Restrict Pod resources to use a known service account can be useful to ensure workload identity. This policy uses a ConfigMap resource as an external data source to map service accounts to images. Example: 'sa-name: ["registry/image-name"]'
+---
+
+## Policy Definition
+<a href="https://github.com/kyverno/policies/raw/main//other/restrict_service_account.yaml" target="-blank">/other/restrict_service_account.yaml</a>
```

Comment:
Doesn't cause a broken link, but two forward slashes between "main" and "other" here.

Reply:
Does cause a broken link here: https://kyverno.io/policies/other/restrict-service-account/

Reply:
The issue was the duplicate policy. The prior file had "-" delimiters instead of "_". I deleted the older version.

See: https://deploy-preview-139--kyverno.netlify.app/policies/other/restrict_service_account/